web_external/templates/widgets/leaderboard.jade

```diff
@@ -4,7 +4,7 @@ table.c-leaderboard-table
       th.c-rank Rank
       th User
       th Title
-      th Date
+      th Date (UTC)
```

Comment:
Changed in https://github.com/girder/covalic/pull/178/commits/071978e540342ef6d4d34a3203c27431d07c58e5